TESTING.md

```diff
@@ -124,6 +124,10 @@ This test suite is based on [BackstopJS](https://garris.github.io/BackstopJS) pl
 
 Documentation available [here](https://github.com/garris/BackstopJS#backstopjs)
 
+### Requirements
+ * A sample Mosaico custom configured template.
```

Comment:
give the path to the page where you configure the templates. give as much as info as you can so that the dev is not left guessing

Reply:
Done.

Reply:
Done.